downstream/modules/terraform-aap/con-terraform-integration-workflows.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="integration-workflows"]
+
+= Integration workflows
+
+[role="_abstract"]
+
+Depending on your existing setup, you can integrate these products from {PlatformNameShort} or from {Terraform}. Migration paths are provided for community users and for migrating from the `cloud.terraform` collection to `hashicorp.terraform`.
+
+== Ansible-initiated workflow
+
+Ansible automation hub collections allow {PlatformNameShort} users to leverage {Terraform}'s provisioning capabilities.
+
+**hashicorp.terraform collection**
+
+This collection provides API integration between {PlatformNameShort} and {Terraform}. This solution works natively with {PlatformNameShort} and reduces setup complexity because it doesn’t require a binary installation and it includes a default execution environment.
```

Comment:
Can we clarify that "Terraform" in these contexts refers to a cloud or enterprise Terraform service and not simply Terraform CLI?